wee_slack.py

```diff
@@ -1860,7 +1860,7 @@ def __init__(self, message_json, team, channel, override_sender=None):
         self.suffix = ''
         self.ts = SlackTS(message_json['ts'])
         text = self.message_json.get('text', '')
```

Comment:
Replace this with `….get('text')` too, then? `.get(key)` will default to None if not found.